remote-api/src/main/java/org/alfresco/rest/api/impl/DownloadsImpl.java

```diff
@@ -87,6 +111,8 @@ public Download createDownloadNode(Download download)
         
         checkNodeIdsReadPermission(zipContentNodeRefs);
         
+        checkArchiveStatus(zipContentNodeRefs, 500);

```

Comment:
Magic number - at minimum should be a constant. We should consider make it a configurable property (ie. add to "repository.properties" and inject into this bean.

Reply:
Thanks, refactoring to global property to be overriden by the connectors

Reply:
Thanks, refactoring to global property to be overriden by the connectors

Reply:
Thanks. I'm not sure connectors need to override (do they) ? Wouldn't a global (ie. system-wide) be sufficient ?